### ISSUE TITLE
Remote selection not properly rendered.

### ISSUE THREAD

**Issue by dmonad** [yjs/y-prosemirror#32]
Description from a user: We just found a weird awareness behavior. If user select text in a client, other client can see the correct awareness with the same range selected, but if user select a node ( which we implemented as prosemirror nodeview ), the other client will see the wrong awareness. Just as above recorded video, if user select the resources node, other user will show the following text is selected.

Basically, the issue can be reproduced by selecting an <hr> element. The remote selection of that cursor information is rendered with a offset, not selecting the target element.

**Comment by dongtc** [comment]
Yes, i found the same issue. https://github.com/yjs/y-prosemirror/blob/25cea84874eace50745fd2433847aabceef92b65/src/plugins/sync-plugin.js#L167

**Comment by dmonad** [comment]
Fixed in y-prosemirror@1.0.4
